README.md

```diff
@@ -1,3 +1,11 @@
+
+# PLEASE NOTE, THIS PROJECT HAS BEEN DEPRECATED
+
+[![No Maintenance Intended](http://unmaintained.tech/badge.svg)](http://unmaintained.tech/)
```

Comment:
I think we can leave this out

Reply:
Sure, was just a copy paste from the [iot-explorer](https://github.com/Azure/iothub-explorer) deprecation notice.  Will remove